blockdevice/util/util.go

```diff
@@ -107,3 +107,8 @@ func PartPath(d string, n int) (string, error) {
 		return filepath.Join("/dev", PartName(d, n)), nil
 	}
 }
+
+// PartPathEncrypted gets encrypted partition path which is mounted into /dev/mapper.
+func PartPathEncrypted(partitionName string) string {
+	return fmt.Sprintf("/dev/mapper/%s_encrypted", partitionName)
```

Comment:
just my 2 cents, but I feel `-` or `.` might look nicer than `_`, as usually there aren't underscores in device names?